### ISSUE TITLE
Improve accessibility of contributor fields in Book Info

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#2436]
> When arriving at an input field that is populated with contributor names, the contributors are not announced to assistive technologies, and the input is announced as blank even when values have been populated. It was possible to reach the contributor names in read mode, however tabbing through the inputs some users may still assume the input is blank. It is also difficult to switch into read mode with NVDA because the arrow keys open the suggestions menu, making it even more challenging to read the content within these fields.

**Comment by greatislander** [comment]
See: https://github.com/pressbooks/pressbooks/issues/2434#issuecomment-955578180
